### DISCUSSION TITLE
Touch Display for older childs

### DISCUSSION THREAD

**Discussion by hailogugo** [MiczFlor/RPi-Jukebox-RFID#1746] [Ideas]
Hello,
I hope you can help me with my idea.

I want to use a 5 or 7" Touchscreen on my box and my daughter see a webpage with only some functions.
The display shows only the following things which can be controll.
- Play, Pause, Stopp, Prev, Next, Vol +, Vol -, Time of the Trackplay, Genral Volumen.
- A secound view will be the library of the folders and subfolders. Here it will be nice if we only display the cover, first of the folder and than the covers which are in the subfolders. If you click on the cover it goes deaper into the path and than play the Track/Stream
- Displaying like it is in this [Project](https://github.com/Thyraz/Sonos-Kids-Controller)

I start with some things but I cannot add more because I do not have the knowledge. I start with the index-lcd.php and at the folder list into it. But know I do not understand what can I prepare or something else. because of no knowledge.

Can somebody help me to makes this project work?

**Comment by PsychoD** [comment]
Hi, 

I had a similar idea as you have, and have already build such phoniebox. I use the future3 release, which works on a small touchscreen, but needs some optimization to be usable for kids. 
Check out this discussion: 
https://github.com/MiczFlor/RPi-Jukebox-RFID/discussions/1450

**Comment by hailogugo** [comment]
How can I close this thread?

**Comment by pabera** [comment]
Discussions threads can't be closed, only issues can.
